src/content/en/fundamentals/engage-and-retain/push-notifications/faq.md

```diff
@@ -0,0 +1,154 @@
+project_path: /web/_project.yaml
```

Comment:
If need be, we can address this after publishing, which doesn't change the fact that I feel pretty strongly about this. I should also have said, that where appropriate, these should be moved, or at least copied, to StackOverflow.